<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 28pt;

</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 168, 167);
font-size: 14pt;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>260</y>
     <width>222</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt; 
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt; 
color: rgb(255, 0, 127);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>180</y>
     <width>222</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14pt; 
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt; 
color: rgb(255, 0, 127);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="dontHaveAcc">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>360</y>
     <width>132</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Don't have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAcc">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>351</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="err_invalidemail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12pt;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Invalid Credentials</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_success">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12pt;
color: rgb(49, 202, 38);</string>
   </property>
   <property name="text">
    <string>Logged in successfully!</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>createAcc</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
